src/js/components/about/MoreInfo.jsx

```diff
@@ -23,9 +23,7 @@ const MoreInfo = () => (
                 </a>
                 &nbsp;and the&nbsp;
                 <a
-                    target="_blank"
-                    rel="noopener noreferrer"
-                    href="https://www.fiscal.treasury.gov/data-transparency/data-act-v1.2.html">
+                    href="https://www.usaspending.gov/#/download_center/data_dictionary">
```

Comment:
Could you also make this a relative link? (`/#/download_center/data_dictionary`) That way you don't get redirected to prod when clicking from local/ dev/ staging. Same for the footer link.